### PR TITLE
New Proto fixed

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) New Protos added.
2) Removed the Cost per block.
3) Updated the examples.
4) Updated the tests.
5) Added the validation if cost per block is not added in the recipe.

### CODE REVIEW

File: lib/src/features/validations/validate_recipe.dart [hunk 1]
```diff
@@ -3,39 +3,32 @@ import 'dart:convert';
...
-
-
-
   /// Verifies that all of a recipe's outputs are accessible, and that it
   /// doesn't refer to any outputs that are not present.
   static void validate(Recipe recipe) {
-
-
-
     if (recipe.name.length <= 8) {
```
Comment (mustafapylons) [inline lib/src/features/validations/validate_recipe.dart:9]: Number should be in a constant

File: lib/src/features/validations/validate_recipe.dart [hunk 2]
```diff
@@ -3,39 +3,32 @@ import 'dart:convert';
...
       throwError(recipe, 'Recipe name should have more than 8 characters');
     }
 
-
-    if (recipe.description.length <= 20 ) {
```
Comment (mustafapylons) [inline lib/src/features/validations/validate_recipe.dart:20]: Please use constant here as well.

File: lib/src/features/validations/validate_recipe.dart [hunk 3]
```diff
@@ -3,39 +3,32 @@ import 'dart:convert';
...
 
-
     if (recipe.iD.isEmpty) {
       throwError(recipe, 'Invalid Recipe ID');
     }
 
-    
-    if(recipe.itemInputs.where((e) => e.iD.isEmpty).toList().isNotEmpty ){
+    if (recipe.itemInputs.where((e) => e.iD.isEmpty).toList().isNotEmpty) {
       throwError(recipe, 'Invalid Item ID');
```
Comment (mustafapylons) [inline lib/src/features/validations/validate_recipe.dart:26]: more meaningful variable than "e".

**Review by mustafapylons — COMMENTED** [review]
Few things to fix, otherwise clean code.

File: lib/src/generated/pylons/payment_info.pb.dart [hunk 4]
```diff
@@ -10,114 +10,42 @@ import 'dart:core' as $core;
...
-      create()..mergeFromJson(i, r);
-  @$core.Deprecated('Using this can add significant overhead to your binary. '
-      'Use [GeneratedMessageGenericExtensions.deepCopy] instead. '
-      'Will be removed in next major version')
+  factory PaymentInfo() => create();
+  factory PaymentInfo.fromBuffer($core.List<$core.int> i, [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) => create()..mergeFromBuffer(i, r);
+  factory PaymentInfo.fromJson($core.String i, [$pb.ExtensionRegistry r = $pb.ExtensionRegistry.EMPTY]) => create()..mergeFromJson(i, r);
+  @$core.Deprecated(
+      'Using this can add significant overhead to your binary. '
+          'Use [GeneratedMessageGenericExtensions.deepCopy] instead. '
```
[the remark this answers is not in the document]
Comment (kjawadDeveloper2) [reply lib/src/generated/pylons/payment_info.pb.dart:29]: @mustafapylons  its auto generated code from the starport.

**Review by mustafapylons — APPROVED** [review]
Looks good to me now.

**Review by afti-githobo — APPROVED** [review]

**Pull request merged**